src/components/home/Home.jsx

```diff
@@ -45,6 +46,16 @@ const Home = ({ route }) => {
       console.error('Error retrieving weather data frontend:', error);
```

Comment:
add getPlaylist function inside of getWeatherData function after response returns